source/Halibut.Tests/Transport/SecureClientFixture.cs

```diff
@@ -39,8 +42,14 @@ public void Dispose()
         }
 
         [Test]
-        public void SecureClientClearsPoolWhenAllConnectionsCorrupt()
+        [SyncAndAsync]
+        public async Task SecureClientClearsPoolWhenAllConnectionsCorrupt(SyncOrAsync syncOrAsync)
         {
+            if (syncOrAsync == SyncOrAsync.Async)
+            {
+                Assert.Inconclusive("TODO - ASYNC ME UP! Code path incomplete for the system under test.");
```

Comment:
For my understanding when does this become async?

Reply:
This has been changed so the test now passes. We now wire up incomplete async paths to the sync code path